Bottom: aec201736a4f68bf24a0feaac19075cd3c2d9ec1
Top:    d52c8fc5bd11120387bedcf4dbc60c56e9234e1f
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-10 12:19:58 -0500

Ignore exim4.conf.template

We don't use this file, so ignore it


---
diff --git a/docker/docker-exim4/docker-entrypoint.sh b/docker/docker-exim4/docker-entrypoint.sh
index 8d9d35cb..d15e26ce 100755
--- a/docker/docker-exim4/docker-entrypoint.sh
+++ b/docker/docker-exim4/docker-entrypoint.sh
@@ -21,7 +21,6 @@ chown -R root:mail /var/mail
 case ${1} in
     exim4)
         if   [[ -f ${CONFIG_PATH}/update-exim4.conf.conf ]] \
-          && [[ -f ${CONFIG_PATH}/exim4.conf.template ]] \
           && [[ ! -f ${CONFIG_PATH}/exim4.conf ]]
         then
             # set the hostname
